src/lib/rest/rest.cpp

```diff
@@ -614,6 +614,65 @@ bool urlCheck(ConnectionInfo* ciP, const std::string& url)
   return true;
 }
 
+/* ****************************************************************************
+*
+* defaultServicePath
+*
+* Returns a default servicePath (to be used in the case Fiware-servicePath header
+* is not found in the request) depending on the RequestCheck
+*/
+std::string defaultServicePath(const char* url, const char* method)
+{
+  /* Look for standard operation (based on the URL). Given that some operations are
+   * a substring of other, we search first for the longest ones
+   *
+   *  updateContextAvailabilitySubscription
+   *  unsubscribeContextAvailability
+   *  subscribeContextAvailability
+   *  discoverContextAvailability
+   *  updateContextSubscription
+   *  unsubscribeContext
+   *  subscribeContext
+   *  registerContext
+   *  updateContext
+   *  queryContext
+   *
+   * Note that in the case of unsubscribeContext and unsubscribeContextAvailability doesn't
+   * actually use a servicePath, but we also provide a default in that case for the sake of
+   * completeness
+   */
+
+  // FIXME P5: this strategy can be improved taking into account full URL. Otherwise, it is
```

Comment:
If you think the function can work in a better way, as suggest this FIXME (you know C-based string comparations techniques better than me :) easy to implemnt, tell me so. Otherwise, mark this with NTC and let it for the future.